### ISSUE TITLE
getBatteryCharge returns null

### ISSUE THREAD

**Issue by lykhin** [andig/fritzapi#4]
getBatteryCharge returns null, while getTemperature returns correct value. Fritz!OS version is 06.83, Fritz!Box software says the battery level is 100%

```
[2017-9-3 13:52:27] [FritzBox] > getTemperature ("ccbc27e116a7967a","119590367688")
[2017-9-3 13:52:28] [FritzBox] < getTemperature 23
[2017-9-3 13:52:28] [FritzBox] > getBatteryCharge ("ccbc27e116a7967a","119590367688")
[2017-9-3 13:52:29] [FritzBox] < getBatteryCharge null
```

<img width="567" alt="screen shot 2017-09-03 at 13 55 08" src="https://user-images.githubusercontent.com/7879041/30002665-9232e360-90af-11e7-84b8-3d67e5314bad.png">

**Comment by andig** [comment]
Battery charge has no API so we rely on screen scraping. See https://github.com/andig/fritzapi/blob/master/index.js#L543- it seems its a language problem (battery vs batterie)?

**Comment by andig** [comment]
Fixed via ccf157bf85e1fe1be905172b1e91316ba40c3aff

**Comment by lykhin** [comment]
Thanks a lot, it works now!
